bitrise.yml

```diff
@@ -137,16 +137,24 @@ workflows:
             #!/usr/bin/env bash
             set -e
             make test-spm ci=true
+    - xcodebuild:
+        title: Build SPMProject for tests - Catalyst
+        inputs:
+        - scheme: SPMProject
+        - destination: platform=macOS,variant=Mac Catalyst
+        - project_path: "$BITRISE_SOURCE_DIR/dependency-manager-tests/spm/SPMProject.xcodeproj"
+        - xcpretty_test_options: --color --report html --output "${BITRISE_DEPLOY_DIR}/SPMProject-catalyst-sanity-check.html"
     - xcode-test:
         title: Run SPMProject tests
         inputs:
         - scheme: SPMProject
+        - simulator_device: iPhone 11
         - is_clean_build: 'yes'
         - cache_level: none
         - project_path: "$BITRISE_SOURCE_DIR/dependency-manager-tests/spm/SPMProject.xcodeproj"
         - xcpretty_test_options: --color --report html --output "${BITRISE_DEPLOY_DIR}/SPMProject-tests.html"
     - script:
-        title: Test Carthage compatibility
+        title: Test Carthage compatibility (Carthage doesn't support Catalyst!)
```

Comment:
The `title` appears in CI logs, so it's IMO better to keep it as code comment:
```
# Carthage doesn't support Catalyst, so we don't test CTProject for `variant=Mac Catalyst`
- script:
   title: Test Carthage compatibility
   ...
```
WDYT?